.github/workflows/push_dict_to_repos.yml

```
@@ -0,0 +1,95 @@
+name: Push dictionaries to OpenNeuro-JSONLD
+
+on:
+  push:
+    branches:
+      - main
+      - add_automation
+  workflow_dispatch:
+
+jobs:
+  update-datasets:
+    runs-on: ubuntu-latest
+    timeout-minutes: 720
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Find and submit data dictionaries
+        id: get_datasets
```

```suggestion
        id: get_dictionaries
```
since not all OpenNeuroDatasets are represented in openneuro-annotations?